x-pack/plugin/ml/cpp-snapshot/build.gradle

```diff
@@ -9,7 +9,7 @@ ext.version = VersionProperties.elasticsearch
 // for this project so it can be used with dependency substitution.
 
 void getZip(File snapshotZip) {
-  String zipUrl = "http://prelert-artifacts.s3.amazonaws.com/maven/org/elasticsearch/ml/ml-cpp/${version}/ml-cpp-${version}.zip"
+  String zipUrl = "http://prelert-artifacts.s3.amazonaws.com/maven/org/elasticsearch/ml/ml-cpp-df/${version}/ml-cpp-df-${version}.zip"
```

Comment:
Think we could log a big warning message out to console to aid in reminding us?

Reply:
@dimitris-athanasiou created an issue for that, so it's being tracked, a log line probably gets lost (this is executed more to the beginning than towards the end).

(BTW The good news is also that it's not possible this leaks into a release, because you would need a non-snapshot build, plus release build servers work differently)